### PR TITLE
RN-429: Break out the LESMIS Admin Panel into a separate app

### PR DESCRIPTION
**Pull request by rohan-bes**
### Issue RN-429:

2 main changes here:
1. Break the LESMIS Admin Panel out into a separate app. This means that the user will log into it separately
2. Add the viz-builder to the LESMIS Admin Panel

Mainly the work here involved passing in custom redirects given that the base urls for the admin panel in LESMIS are different to the standard admin panel.

Merge tasks:
- [ ] Add `REACT_APP_VIZ_BUILDER_API_URL` .env var to `lesmis` and `admin-panel` packages
- [ ] Add `REACT_APP_ADMIN_API_URL` .env var to `lesmis`

### CODE REVIEW

File: packages/admin-panel/src/index.js
Comment (rohan-bes) [inline packages/admin-panel/src/index.js:34]: Pushed all the themes and css stuff up to this level, so that they wouldn't clash with the LESMIS ones when we import there.

File: packages/lesmis/src/routes/AdminPanelApp.js
Comment (rohan-bes) [inline packages/lesmis/src/routes/AdminPanelApp.js:180]: Once @chris-pollard's changes come in we won't need this `isBESAdmin` stuff, so not really worried about it here 👍

**Review by billli0 — COMMENTED** [review]
Just got a few questions before approval, but overall it looks great!

**Comment by billli0** [comment]
Got a bug when hitting `New` on Visualisations->Dashboard Items tab. It will redirect to the home page.

**Comment by rohan-bes** [comment]
>Got a bug when hitting New on Visualisations->Dashboard Items tab. It will redirect to the home page.

Ah sorry @billli0. There's a few .env vars that have to be added to get this feature to work properly:

admin-panel:
`REACT_APP_VIZ_BUILDER_API_URL="https://[deployment-name]-admin-api.tupaia.org/v1"`

lesmis package:
REACT_APP_ADMIN_API_URL="https://[deployment-name]-admin-api.tupaia.org/v1"
REACT_APP_VIZ_BUILDER_API_URL="https://[deployment-name]-admin-api.tupaia.org/v1"

Or `http://localhost:8070/v1` for all of theses 👍

File: packages/devops/configs/servers.conf
Comment (rohan-bes) [reply packages/devops/configs/servers.conf:239]: I'm not sure... but I don't think so. I copied this from the same config for `api.lesmis.la`

File: packages/lesmis/src/routes/AdminPanelApp.js
Comment (rohan-bes) [reply packages/lesmis/src/routes/AdminPanelApp.js]: Good point, I think I had it from the ones created from the `routes` array, but it's not needed here so I'll remove it 👍

**Review by billli0 — APPROVED** [review]
LGTM, just one request to remove 'key={`${adminUrl}/viz-builder`}'.

**Pull request merged**
